src/cljs/rems/spa.cljs

```diff
@@ -31,7 +32,8 @@
 
 (defn home-page []
   (if @(rf/subscribe [:user])
-    [:p "Logged in."]
+    (do (rf/dispatch [:landing-page-redirect!]) ; don't show the empty page
+          [:div])
```

Comment:
couldn't this just `dispatch!` to `/landing_page`? or actually, could we get rid of landing pages altogether and have the logic in `/`?